### ISSUE TITLE
ENH add simple example_config.yml to get started

### ISSUE THREAD

**Issue by mathurinm** [benchopt/benchmark_lasso#80]
We can commit a simple config with a small dataset, few objectives, some fast solvers (sklearn, celer, cd ?) and add a snippet in the Readme so that people can get here, clone the repo, and get a benchmark in 1 min

May @Klopfe @QB3 or @tbng would have the time to do it ?

**Comment by tbng** [comment]
You mean like this @mathurinm : 

```yaml
# example_config.yml
dataset: # dataset to run benchmark on
  - leukemia
  - simulated

force-solver: # list of example solvers to do benchmark
  - celer
  - cd
  - sklearn

# specify more than 1 repeated runs to have error-bar on the plot
n-repetitions: 1
```
and add to README:
```markdown
# Quick Start

To perform a quick test benchmark, one can run:

`benchopt run --config example_config.yml`

This config file will perform benchmarks of `celer`, `cd` (coordinate-descent), and `sklearn` implementation of the sparse solver on 2 dataset: leukemia and toy (simuldated dataset)
```
Or of course to quick test the library with good plots one can reduce to much smaller dataset and get faster results

**Comment by mathurinm** [comment]
Yes exactly like this ! Can you send a PR ?

Once https://github.com/benchopt/benchopt/pull/362/ we can improve by creating a small `simulated` dataset for this purpose !

**Comment by mathurinm** [comment]
Solved in #82
